src/components/panes/EditCallerPane.jsx

```diff
@@ -74,18 +74,18 @@ export default class EditCallerPane extends PaneBase {
                 onAdd={ this.onAddTag.bind(this, 'ex') }
                 onRemove={ this.onRemoveExTag.bind(this) }/>
         }
-
+        
         return [
             <h3 key="prioHeader">
-                <i className="fa fa-star"/>
-                <Msg id="panes.editCaller.prio.h"/>,
+                <i className="fa fa-star" style = {{padding : "10px"}}/>
+                <Msg id="panes.editCaller.prio.h"/>
```

Comment:
Nice catch on the comma, which for some reason I must have missed when I first created this pane, and which has been sitting there for years by now. 🤦